tests/core/test_utils.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+from a5.core.utils import PentagonShape, Contour, Pentagon
+from a5.core.coordinate_systems import Degrees, LonLat
+
+# Create a simple pentagon for testing
+@pytest.fixture
```

Comment:
The tests are a bit light here, I will try and add more in the JS version so we can be sure the things like `rotate180` work